pkg/virt-handler/cache/cache_test.go

```diff
@@ -389,6 +397,8 @@ var _ = Describe("Domain informer", func() {
 
 			domain := api.NewMinimalDomain("test")
 			list = append(list, domain)
+			err := ghostRecordStore.Add("default", "test", socketPath, podUID)
```

Comment:
@enp0s3 why aren't you using something like `DeferCleanup` in these tests? or at least deleting these reconrds?
Wouldn't there be a race if the records are not deleted in time?

Reply:
yes, but I would still remove these or use the DeferCleanup or explicitly cleanup in AfterEach.